test/e2e/operator_test.go

```diff
@@ -700,6 +701,120 @@ func TestTLSSecurityProfile(t *testing.T) {
 	}
 }
 
+func TestRouteAdmissionPolicy(t *testing.T) {
+	// Set up an ingresscontroller which only selects routes created by this test
```

Comment:
I would have expected this sort of test to be  added in openshift/router (and I'm surprised we don't already have something in openshift/origin) and for the operator's test only to verify that the deployment has the correct configuration, but I suppose adding the test to cluster-ingress-operator is fine (and it is a very fine test indeed!).

Reply:
Hmmm... good point — I can move the e2e to openshift/origin and replace this one with a unit test that verifies the router config. Maybe I could do next in a followup?